src/pages/location/createLocation/components/AddZoneLocationDialog.ts

```diff
@@ -0,0 +1,13 @@
+import BasePageModel from '@/pages/BasePageModel';
+
+class AddZoneLocationDialog extends BasePageModel {
+  get zoneLocationNameField() {
```

Comment:
Let's add a `TextField` property for this instead.

Reply:
```
`get textbox() {
    return this.field.getByRole('textbox');
  }`
```

```
get field() {
    return this.root.locator(
      `div[data-testid="form-field"][aria-label="${this.fieldName}"]`
    );
  }
```
We can't use `TextField` here as this field doesn't have data-testid and aria-label.